packages/jadmin/jadmin.js

```diff
@@ -330,5 +330,78 @@ yargs
       await db.pool.end()
     },
   })
+  .command({
+    command: 'assign-badge <userId> <badgeName>',
+    describe: 'Assign a badge to a user',
+    builder: (yargs) => {
+      yargs.option('force', {
+        describe: 'Bypass interactive safety checks',
+        type: 'boolean',
+      })
+      yargs.option('database-url', {
+        describe: 'The URL to connect to the database',
+        type: 'string',
+      })
+    },
+    handler: async (args) => {
+      const dbUrl = args.databaseUrl || process.env.DATABASE_URL
+
+      if (!dbUrl) {
+        console.error('No database URL has been provided')
+        process.exit(1)
+      }
+
+      const parsedDbUrl = parse(dbUrl)
+      let promptResponse
+
+      if (!['localhost', '127.0.0.1'].includes(parsedDbUrl.host) && !args.force) {
+        promptResponse = await prompts({
+          type: 'confirm',
+          name: 'confirmDbUrl',
+          message: `CAUTION: your $DATABASE_URL is currently set to a remote database: ${dbUrl}. Are you sure you want to continue?`,
+        })
+
+        if (!promptResponse.confirmDbUrl) return
+      }
+
+      const db = pgTag(
+        new Pool({
+          connectionString: dbUrl,
+        }),
+      )
```

Comment:
We do this DB check in all three scripts now I think, and I think it's exactly the same check, so this would be great to factor out:

```suggestion
    const async getDb = (args) => {
      const dbUrl = args.databaseUrl || process.env.DATABASE_URL

      if (!dbUrl) {
        console.error('No database URL has been provided')
        process.exit(1)
      }

      const parsedDbUrl = parse(dbUrl)
      let promptResponse

      if (!['localhost', '127.0.0.1'].includes(parsedDbUrl.host) && !args.force) {
        promptResponse = await prompts({
          type: 'confirm',
          name: 'confirmDbUrl',
          message: `CAUTION: your $DATABASE_URL is currently set to a remote database: ${dbUrl}. Are you sure you want to continue?`,
        })

        if (!promptResponse.confirmDbUrl) return
      }

      const db = pgTag(
        new Pool({
          connectionString: dbUrl,
        }),
      )
      
      return db
    }
    
    ... inside handlers ...
    
    const db = await getDb(args)
```

Reply:
Great call! Done ✅✏️🦔